docs: just cleanup following JCBs edits (#3976)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/cleanup/index.md

If Gatekeeper is installed on the cluster, the upgrade leaves a Helm Secret, Deployments, and a Service that belonged to the 1.8 Gatekeeper addon in the `kubeaddons` namespace.

> <p class="message--note"><strong>WARNING: </strong>As long as the secret exists, <code>helm list -n kubeaddons</code> incorrectly reports the existence of a "gatekeeper-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it might corrupt the Gatekeeper Platform Application in Kommander, potentially making Kubernetes on the cluster inoperable.</p>

Reviewer: ```suggestion
<p class="message--note"><strong>WARNING: </strong>As long as the secret exists, <code>helm list -n kubeaddons</code> incorrectly reports the existence of a "gatekeeper-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it may corrupt the Gatekeeper Platform Application in Kommander, making Kubernetes inoperable on the cluster.</p>
```

Delete these objects using the `kubectl`commands:

```sh
kubectl delete -n kubeaddons secret sh.helm.release.v1.gatekeeper-kubeaddons.v1
kubectl delete -n kubeaddons deployments gatekeeper-audit gatekeeper-controller-manager
kubectl delete -n kubeaddons service gatekeeper-webhook-service
```

<p class="message--note"><strong>WARNING: </strong>As long as the secret exists, <code>helm list -n kubeaddons</code> wrongly reports the existence of a "gatekeeper-kubeaddons" Helm release.

> If you manipulate this "Helm release" using Helm, it might corrupt the Gatekeeper Platform Application in Kommander, potentially making Kubernetes on the cluster inoperable.</p>

Reviewer: ```suggestion
If you manipulate this "Helm release" using Helm, it may corrupt the Gatekeeper Platform Application in Kommander, making Kubernetes inoperable on the cluster.</p>
```
